### ISSUE TITLE
Installing protorl as local package dependency conflict error

### ISSUE THREAD

**Issue by ktvalverde** [philtabor/ProtoRL#7]
When following the "Recommended Setup" section of the readme file, I get dependency conflicts with shimmy[atari] and ale-py.

`pip install .`

![protorl dependency conflicts](https://github.com/user-attachments/assets/062cb17e-74e2-48f2-8f4b-0f67c74647e5)

**Steps to recreate.**

1. Clone protorl repository
2. Setup new virtual environment
3. Run: pip install .

**Comment by philtabor** [comment]
I can't seem to recreate this issue. I end up with ale 0.8.1 and shimmy 0.2.1.

The only thing I can think of is perhaps your cache needs to be purged?

**Comment by ktvalverde** [comment]
Right, could be. I went ahead and started up a fresh, new Python virtual environment outside of an IDE and with a clean install of python 3.8. Now I am getting new dependency errors when running "pip install ."
Not sure what the issue is here but I am on a Windows machine. I will try Linux later tonight.

**Comment by ktvalverde** [comment]
I couldn't recreate the issue in any other instance I tried. I am still struggling with different dependency issues but I will close this issue as I can't recreate it.

**Comment by philtabor** [comment]
I'm reopening this issue: I've done a reformat on my system and I now have the same problem. I believe this is related to the Python version. Kubuntu has defaulted to 3.12, while ale-py seems to require a lower version. 

I will figure out the newest version of python to work, and then update the setup file accordingly.
